packages/@okta/vuepress-theme-prose/assets/css/abstracts/_colors.scss

```diff
@@ -8,6 +8,34 @@ $colors: (
     'base': #E22866,
     'active':#dd2864,
   ),
+  // redesign 2021-07
+  'header': (
+    'base': #00297A,
+  ),
+  'body': (
+    'base': #F5F5F6,
+    'border': #DCDCDC,
+  ),
+  'text': (
+    'darker': #1D1D21,
+    'base': #6E6E78,
+    'lighter': #D7D7DC,
+  ),
+  'link': (
+    'base': #1662DD,
+    'darker': #00297A,
+    'footer': #B5D0DF,
+  ),
+  'code': (
+    'keyword': #965ABE,
+    'define': #F19B56,
+    'property': #8BC5EB,
+    'string': #C4C4C4,
+  ),
+  'form': (
+    'input-border': #C1C1C8,
+  ),
+
```

Comment:
Why are we hardcoding more color values? Design tokens like color should be imported from Odyssey: https://odyssey.okta.design/base/tokens/

Reply:
I never heard of the odyssey. Can you please point in the code where this is used? All I can find is   
// Copied from Odyssey 
comment...